### PR TITLE
magento/devdocs#: Add createCustomer mutation API errors

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds information about API errors of `createCustomer` mutation.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/graphql/mutations/create-customer.html

## Links to Magento source code

-  [A customer with the same email address already exists in an associated website.](https://github.com/magento/magento2/blob/2.3.3/app/code/Magento/Customer/Model/ResourceModel/Customer.php#L142)
-  ["Email" is not a valid email address.](https://github.com/magento/graphql-ce/blob/2.3.3/app/code/Magento/Eav/Model/Attribute/Data/AbstractData.php#L380)
-  [Field CustomerInput.email of required type String! was not provided](https://github.com/webonyx/graphql-php/blob/master/src/Validator/Rules/ValuesOfCorrectType.php#L261)
-  [Field "xxx" is not defined by type CustomerInput.](https://github.com/webonyx/graphql-php/blob/master/src/Validator/Rules/ValuesOfCorrectType.php#L266)
-  [Required parameters are missing: First Name](https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/CustomerGraphQl/Model/Customer/ValidateCustomerData.php#L59)

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/mutations/create-customer.md [hunk 1]
```diff
@@ -69,6 +69,16 @@ The `createCustomer` mutation returns the `CustomerOutput` object.
 
 {% include graphql/customer-output.md %}
 
+## Errors
+
+Error | Description
+--- | ---
+`A customer with the same email address already exists in an associated website.` | Customer with a provided email in the `input`.`email` already exists.
```
Comment (keharper) [inline guides/v2.3/graphql/mutations/create-customer.md]:
```suggestion
`A customer with the same email address already exists in an associated website.` | The email provided in the `input`.`email` attribute belongs to an existing customer.
```

File: guides/v2.3/graphql/mutations/create-customer.md [hunk 2]
```diff
@@ -69,6 +69,16 @@ The `createCustomer` mutation returns the `CustomerOutput` object.
...
+`"Email" is not a valid email address.` | Value provided in the `input`.`email` has invalid email format.
```
Comment (keharper) [inline guides/v2.3/graphql/mutations/create-customer.md]:
```suggestion
`"Email" is not a valid email address.` | The value provided in the `input`.`email` attribute has an invalid format.
```

File: guides/v2.3/graphql/mutations/create-customer.md [hunk 3]
```diff
@@ -69,6 +69,16 @@ The `createCustomer` mutation returns the `CustomerOutput` object.
...
+`Field CustomerInput.email of required type String! was not provided` | The `input`.`email` attribute was omitted.
+`Field "xxx" is not defined by type CustomerInput.` | The `input`.`xxx` is undefined attribute.
```
Comment (keharper) [inline guides/v2.3/graphql/mutations/create-customer.md]:
```suggestion
`Field "xxx" is not defined by type CustomerInput.` | The `input`.`xxx` is an undefined attribute.
```

**Comment by rogyar** [comment]
Hi @atwixfirster. There's one more possible [error](https://github.com/magento/magento2/blob/33d81489258c765ba6d5f4c6e4f8c5f78c835f21/app/code/Magento/CustomerGraphQl/Model/Resolver/CreateCustomer.php#L67):

```
'"input" value should be specified'
```

**Comment by atwixfirster** [comment]
> Hi @atwixfirster. There's one more possible [error](https://github.com/magento/magento2/blob/33d81489258c765ba6d5f4c6e4f8c5f78c835f21/app/code/Magento/CustomerGraphQl/Model/Resolver/CreateCustomer.php#L67):
> 
> ```
> '"input" value should be specified'
> ```

Good catch but no. Please check - https://github.com/magento/graphql-ce/pull/1052

Thank you, @rogyar

File: guides/v2.3/graphql/mutations/create-customer.md (hunk 3 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/mutations/create-customer.md]: **fixed**

Thanks, @keharper

File: guides/v2.3/graphql/mutations/create-customer.md (hunk 2 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/mutations/create-customer.md]: **fixed**

File: guides/v2.3/graphql/mutations/create-customer.md (hunk 1 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/mutations/create-customer.md]: **fixed**

**Comment by rogyar** [comment]
That makes sense, thank you

**Comment by keharper** [comment]
running tests

**Pull request merged**